src/client/request_sender.cpp

```diff
@@ -101,10 +100,6 @@ int RequestSender::ReadChunk(const ChunkIDInfo& idinfo,
         request.set_clonefileoffset(sourceInfo.cloneFileOffset);
     }
 
-    if (iosenderopt_.chunkserverEnableAppliedIndexRead && appliedindex > 0) {
```

Comment:
you can remove `chunkserverEnableAppliedIndexRead` from the option and remove related conf in conf/client.conf